Makefile

```diff
@@ -21,18 +21,21 @@ TEST_JAR=test.jar
 CC ?= gcc
 CXX ?= g++
 STRIP ?= strip
+OBJCOPY ?= objcopy
 
 ifneq ($(CROSS_COMPILE),)
 CC := $(CROSS_COMPILE)gcc
 CXX := $(CROSS_COMPILE)g++
 STRIP := $(CROSS_COMPILE)strip
+OBJCOPY := $(CROSS_COMPILE)objcopy
 endif
 
 CFLAGS_EXTRA ?=
 CXXFLAGS_EXTRA ?=
 CFLAGS=-O3 -fno-exceptions $(CFLAGS_EXTRA)
-CXXFLAGS=-O3 -fno-exceptions -fno-omit-frame-pointer -fvisibility=hidden -std=c++11 $(CXXFLAGS_EXTRA)
 CPPFLAGS=
+CXXFLAGS=-O3 -fno-exceptions -fno-omit-frame-pointer -fvisibility=hidden -std=c++11 $(CXXFLAGS_EXTRA)
+LIB_PROFILER_CXXFLAGS=$(CXXFLAGS)
```

Comment:
We already have many variables with flags, can we do without another one?
Can't we just add an option to CXXFLAGS conditionally?